src/schema/submission.yaml

```diff
@@ -1443,13 +1470,13 @@ slots:
   reference_genome:
     description: >-
       A published genetic sequence that is used as a reference sequence against which other sequences are compared.
-      Reference genome(s) or annotation(s) used for prior analyses (eg: GRCh38.p13).
+      Reference genome(s) or annotation(s) used for prior analyses, e.g. GRCh38.p13.
```

Comment:
Is it a multi-valued field?

Reply:
No, I updated the description accordingly.